L1Trigger/L1TCalorimeter/interface/AccumulatingSort.h

```diff
@@ -0,0 +1,50 @@
+#include <vector>
+#include <list>
+#include <stdint.h>
+
+
+template < typename T >
+class AccumulatingSort
+{
+
+private:
+    std::vector < std::list < T > > mSortArrays;
+    uint32_t mSize;
+
+private:
+    void AccumulatorUnit( const std::list< T >& aInput , T& aAcc , std::list< T >& aTail )
+    {
+        aTail.clear();
+
+        bool lAccInserted( false );
+
+        for( typename std::list< T >::const_iterator lIt( aInput.begin() ) ; lIt != aInput.end() ; ++lIt )
```

Comment:
@davidlange6 Since lists don't have <reserve> method, I am not sure what exactly you meant here with reserving the memory.  Method  <resize> would value-initilize the list. Did you have something particular in mind?